core/templates/pages/contributor-dashboard-page/services/contribution-and-review.service.spec.ts

```diff
@@ -287,7 +287,8 @@ describe('Contribution and review service', () => {
         fetchSuggestionsAsyncSpy.and.returnValue(
           Promise.resolve(backendFetchResponse));
 
-        cars.getReviewableTranslationSuggestionsAsync()
+        cars.getReviewableTranslationSuggestionsAsync(
+          /* ShouldResetOffset= */ true, 'skill_id_1')
```

Comment:
Why is this commented?

Reply:
It's a coding practice to comment the name of boolean parameters to make it clearer what value is being set.

Reply:
I think this makes sense but it is not really used in our codebase. Can we maybe move the boolean as a second parameter and the keep the named param without the comment?

Reply:
AFAIK TypeScript doesn't support named parameters—we'd have to declare an interface for the parameters to obtain similar functionality.